integration/env_test.go

```diff
@@ -268,3 +268,14 @@ func (e env) Home() string {
 	require.NoError(e.t, err)
 	return home
 }
+
+// AppHome returns appd's home dir.
+func (e env) AppdHome(name, sdkVersion string) string {
+	switch sdkVersion {
+	case Stargate:
+		return filepath.Join(e.Home(), fmt.Sprintf("%sd", name))
```

Comment:
Does it still has this format for `gaia`? I though it was `~/.app/`

Reply:
Yep, since Stargate the home folder for app is not hidden anymore.